tests/test_enterprise/views/test_course_enrollment_view.py

```diff
@@ -205,6 +205,81 @@ def test_get_course_enrollment_page(
         response = self.client.get(enterprise_landing_page_url)
         self._check_expected_enrollment_page(response, expected_context)
 
+    @mock.patch('enterprise.views.render', side_effect=fake_render)
+    @mock.patch('enterprise.api_client.lms.embargo_api')
+    @mock.patch('enterprise.views.CourseCatalogApiServiceClient')
+    @mock.patch('enterprise.views.EnrollmentApiClient')
+    @mock.patch('enterprise.api_client.ecommerce.ecommerce_api_client')
+    @mock.patch('enterprise.utils.Registry')
+    def test_hide_course_original_price_value_on_enrollment_page(
```

Comment:
Please combine both tests by using ddt for default value of `hide_course_original_price` in `EnterpriseCustomerFactory`.